src/server.ts

```diff
@@ -26,7 +26,16 @@ app.use(
 );
 
 // Handle logs in console during development
-app.use(morgan('dev'));
+app.use(
+  morgan('dev', {
+    skip: (req, res) => {
+      // logs everything but health checks on dev, errors only otherwise
+      return process.env.NODE_ENV === 'development' || process.env.DEBUG === 'true'
+        ? req.originalUrl.includes(HEALTH_ENDPOINT)
+        : res.statusCode < 400;
```

Comment:
restored the specified logging, as it's going to get pretty loud while debugging in cluster.